core/mempool/src/lib.rs

```diff
@@ -419,7 +419,7 @@ pub enum MemPoolError {
     InsertCandidate { len: usize },
 
     #[display(fmt = "Tx: {:?} check_sig failed", tx_hash)]
-    CheckSig { tx_hash: Hash },
+    CheckAuthorization { tx_hash: Hash },
```

Comment:
if you change to `CheckAuthorization` that error message should be changed as well.